jitsi/install

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+sleep 30
+echo 'deb https://download.jitsi.org stable/' | sudo tee -a /etc/apt/sources.list.d/jitsi-stable.list
+sudo wget -qO -  https://download.jitsi.org/jitsi-key.gpg.key | sudo apt-key add -
+sudo apt-get install apt-transport-https debconf-utils
```

Comment:
We may need to run the `sudo apt-get update` before we do the `sudo apt-get install` in order for it to find the debconf-utils package.